src/common/components/userMatchModal/userMatchModal.tpl.html

```diff
@@ -38,10 +38,13 @@
       <button type="button" class="btn btn-md btn-block btn-primary" ng-click="$ctrl.onSuccess()" translate>Close</button>
     </p>
   </div>
-  <div ng-switch-when="success-failure" ng-init="$ctrl.analyticsFactory.track('aa-registration-match-new-account-success')">
-    <p translate>Your account is now activated.</p>
+  <div ng-switch-when="failure" ng-init="$ctrl.analyticsFactory.track('aa-registration-match-failed-questions')">
+    <p translate>
+      Unfortunately we are unable to verify your account.
+      Please contact us at 888-278-7233, option 1, option 1 or email eGift@cru.org.
```

Comment:
Why is this content duplicated in 2 places? I don't quite remember how these modals work...

Reply:
I just happened to use the same text in the failed verification modal as I did here. This is the text supplied by the stakeholders upon failure of the questions. This location is just text on an existing modal, but the other I had to create a new modal or hijack the register account modal, which seemed weird to me.

Reply:
Might be able to use the new directive instead?

Reply:
I don't have my head wrapped around this... The state changes are weird.